filament/backend/CMakeLists.txt

```diff
@@ -450,3 +450,19 @@ if (APPLE AND NOT IOS)
     set_target_properties(backend_test_mac PROPERTIES FOLDER Tests)
 endif()
 
+# ==================================================================================================
+# Compute tests
+
+add_executable(compute_test
```

Comment:
You may need to surround this with a `if (APPLE AND NOT IOS)` block to avoid building on iOS.